src/component/form/CaseElement.tsx

```diff
@@ -249,24 +399,63 @@ export default class CaseElement extends React.Component<CaseElementProps, CaseE
         return errors.length === 0;
     }
 
-    private getArgsEle(): JSX.Element {
-        if (this.state.operatorConfig && this.state.operatorConfig.operands > 0) {
-            const value = this.state.arguments.length ? this.state.arguments[0] : '';
-            const hasArgError: boolean = hasErrorType(this.state.errors, ['argument', 'rules']);
-
-            return (
-                <TextInputElement
-                    data-spec="args-input"
-                    name="arguments"
-                    onChange={this.onChangeArguments}
-                    value={value}
-                    focus={this.props.focusArgsInput}
-                    autocomplete={true}
-                    ComponentMap={this.props.ComponentMap}
-                    showInvalid={hasArgError}
-                    config={this.props.config || null}
-                />
-            );
+    private getArgs(): JSX.Element {
+        if (this.state.operatorConfig.operands > 0) {
+            // First pass at displaying, handling 'has_number_between' inputs
+            if (this.state.operatorConfig.operands > 1) {
+                const { min: minVal, max: maxVal } = getMinMax(this.props.kase.arguments);
+                return (
+                    <React.Fragment>
+                        <TextInputElement
+                            name="arguments"
+                            onChange={this.onChangeMin}
+                            value={minVal}
+                            focus={this.props.focusMin}
+                            ComponentMap={this.props.ComponentMap}
+                            showInvalid={hasErrorType(this.state.errors, [
+                                /Minimum value must/,
+                                /argument/,
+                                /rules/,
+                                /equal/,
+                                /more/
```

Comment:
I know this bit isn't new, but this feels a bit kludgy to me. I think we want to have a think about how to make this more explicit in a future task.

Reply:
yeah agree this is pretty bad. eventually we're going to want real-time error updates like the current editor, right? if so I think the solution will make itself clear as we refactor for that.